src/runtime-rs/crates/agent/src/lib.rs

```diff
@@ -94,4 +94,6 @@ pub trait Agent: AgentManager + HealthService + Send + Sync {
     async fn get_volume_stats(&self, req: VolumeStatsRequest) -> Result<VolumeStatsResponse>;
     async fn resize_volume(&self, req: ResizeVolumeRequest) -> Result<Empty>;
     async fn get_guest_details(&self, req: GetGuestDetailsRequest) -> Result<GuestDetailsResponse>;
+    async fn add_swap(&self, req: AddSwapRequest) -> Result<Empty>;
+    async fn add_swap_path(&self, req: AddSwapPathRequest) -> Result<Empty>;
```

Comment:
What's about just adding one new trait to handle both of the two cases ?

Reply:
add_swap is still used by kata runtime-go. 
So I suggest add a new add_swap_path.